docs/conf.py

```diff
@@ -59,9 +59,7 @@
 # Tell sphinx what the pygments highlight language should be.
 highlight_language = "py"
 
-mathjax_path = (
-    "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
-)
+mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
```

Comment:
The Python formatter does not agree with this change: 🤔
```console
$ python -m black --check --diff --color -l 100 docs/conf.py 
--- conf.py     2022-06-01 19:07:25.506733 +0000
+++ conf.py     2022-06-01 19:09:37.925647 +0000
@@ -57,11 +57,13 @@
 primary_domain = "py"
 
 # Tell sphinx what the pygments highlight language should be.
 highlight_language = "py"
 
-mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
+mathjax_path = (
+    "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
+)
 nbsphinx_requirejs_path = ""
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
@@ -239,13 +241,11 @@
 # html_search_scorer = 'scorer.js'
 
 # Output file base name for HTML help builder.
 htmlhelp_basename = "XanaduCloudClientdoc"
 
-inheritance_node_attrs = dict(
-    color="lightskyblue1", fillcolor="lightskyblue1", style="filled"
-)
+inheritance_node_attrs = dict(color="lightskyblue1", fillcolor="lightskyblue1", style="filled")
 
 # -- Xanadu theme ---------------------------------------------------------
 html_theme = "xanadu"
 
 # Xanadu theme options (see theme.conf for more information).
would reformat conf.py

Oh no! 💥 💔 💥
1 file would be reformatted.
```

You should be able to reproduce this by installing `requirements.txt` and `requirements-dev.txt`.